release_build_versions.txt

```diff
@@ -23,9 +23,13 @@ wasmcloud-1.1.1
 
 tailscale-1.70.0
 
-k3s-v1.29.2+k3s1
+k3s-v1.31.1+k3s1
```

Comment:
hello, if you can put the versions in ascending order, like the other ones.

Reply:
Ah, sorry. Fixed.